scripts/scil_compute_mean_afd_map_from_streamlines.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute the mean Apparent Fiber Density (AFD) and mean Radial fODF (radfODF)
+maps along a bundle.
+
+Can be used to generate AFD and radfODF profiles along a bundle.
+"""
+
+import argparse
+
+import nibabel as nb
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg, add_sh_basis_args,
+                             add_reference_arg,
+                             assert_inputs_exist, assert_outputs_exist)
+from scilpy.reconst.afd_along_streamlines import afd_map_along_streamlines
+
+EPILOG = """
+Reference:
+    [1] Raffelt, D., Tournier, JD., Rose, S., Ridgway, GR., Henderson, R.,
+        Crozier, S., Salvado, O., & Connelly, A. (2012).
+        Apparent Fibre Density: a novel measure for the analysis of
+        diffusion-weighted magnetic resonance images. NeuroImage, 59(4), 3976--3994.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_tractogram',
```

Comment:
Is this script made mostly (only) for bundles and not whole brain?

@mdesco should we phrase the docstring, argument and name of the script so we promote using bundles rather than tractogram?

Reply:
@mdesco I changed the documentation, I'm open to suggestions to rename the script.

Reply:
Thanks @arnaudbore for changing tractogram for bundles. This is good. As for the name of the script, we could change it to 
scil_compute_mean_afd_map_from_bundles.py 

Also, as this is our only, fixel-based script, maybe we should be more explicit:
scil_compute_mean_fixel_afd_map_from_bundles.py
or
scil_compute_mean_fixel_afd_from_bundles.py

I'm good with all three.